lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -467,7 +467,11 @@ def _get_urls(self):  # lint-amnesty, pylint: disable=missing-function-docstring
             self.course_key,
             self.section2.location
         )
-        preview_url = "http://" + settings.FEATURES.get('PREVIEW_LMS_BASE') + lms_url
+        preview_url = '{}/preview/course/{}/{}?foo=b%24r'.format(
+            settings.LEARNING_MICROFRONTEND_URL,
+            self.course_key,
+            self.section2.location
```

Comment:
Does this need to be aware of environments that don't use the Learning MFE? Is that a use case we support?

Reply:
We only support the use of the learning MFE.